php/php.api.phpmodule/src/org/netbeans/modules/php/api/PhpVersion.java

```diff
@@ -35,7 +35,8 @@
     "PhpVersion.PHP_71=PHP 7.1",
     "PhpVersion.PHP_72=PHP 7.2",
     "PhpVersion.PHP_73=PHP 7.3",
-    "PhpVersion.PHP_74=PHP 7.4"
+    "PhpVersion.PHP_74=PHP 7.4",
+    "PhpVersion.PHP_80=PHP 8.0"
```

Comment:
Cannot the trailing comma be added? That would make the diffs much cleaner (next time).

Reply:
I'll fix. Thanks!

Reply:
Fixed: https://github.com/apache/netbeans/compare/63a46a1b8bc87584f63a0d719b73013e8a583887..31c86e7ab8e9dc3a904b71df620023a7f86ac790